department-of-veterans-affairs/vets-website
# MHV-53366 Add consult result notes

## Summary

- Added Consult Result notes to the types of notes we handle (handled identically to Progress Notes)

## Related issue(s)

### [MHV-53366](https://jira.devops.va.gov/browse/MHV-53366) - Ensure we are handling Consultation Notes ###

> Ensure we are handling Consultation Notes
> 
> The MHV FHIR supports (or will support) three note types:
> 
> DS - LOINC#18842-5 "Discharge summary"
> PN - LOINC#11506-3 "Progress note"
> CR - Consult Results – TODO
> us-core LOINC#11488-4 "Consultation Note"
> 
> Verify that CRs are being supported by the backend.
> 
> Ensure we are accepting the LOINC code for Consultation Notes in the backend and that the note is being handled properly on the frontend.
> 
> 
> 
> 
> User Story: As a Medical Records User, I want to  see my consult Notes or results in the UI.
> 
> GIVEN
> 
> WHEN
> 
> THEN
> 
> 
> 
> 
> Feature Flag Y/N ? N
> 
> Manual Testing Y/N ? Y-Sanjay
> 
> Automated Testing Y/N ? Y-Timothy
> 
> Accessibility Testing Y/N ? N
> 
> UCD Validation Y/N N
> 
> 
> 
> 
> Acceptance Criteria:
> 
> AC1  Verify that CRs are being supported by the backend.
> 
> AC2 Ensure we are accepting the LOINC code for Consultation Notes in the backend
> 
> AC3 Verify the note is being handled properly on the frontend.
> 
> AC4
> AC5
> 
> 
> 
> 
> Links to UCD:
> 
> Sketch Link:
> 
> User Flow:
> 
> Mobile:
> 
> Desktop:
> 
> Other Design Notes:
> 
> 
> 
> 
> Architectural Notes: Always check for the latest designs in Figma
> 
> 
> 

## Testing done

- Added unit tests to ensure that CR notes are being routed to the right handler functions

## Screenshots

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/2ab02755-e752-4d31-9527-aa27ed448c68)
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/991aa526-2c30-4d4d-a9ca-505d36a41ef2)


## What areas of the site does it impact?

MHV Medical Records

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
